lib/coreboot/coreboot_table.c

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2017, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <console.h>
+#include <coreboot/coreboot_table.h>
+#include <debug.h>
+#include <mmio.h>
+#include <string.h>
+
+/*
+ * Structures describing coreboot's in-memory descriptor tables. See
+ * <coreboot>/src/commonlib/include/commonlib/coreboot_tables.h for
+ * canonical implementation.
+ */
+
+struct cb_header {
```

Comment:
Yes - I got my terms mixed-up; my comments should have been that structs / enums are typdef'ed.